src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -148,31 +149,31 @@ Info:
     <string name="revanced_hide_playlist_card_shelf_title">Hide playlist card shelf</string>
-    <string name="revanced_hide_playlist_card_shelf_summary">Hides playlist card shelf in feed.</string>
-    <string name="revanced_hide_samples_shelf_title">Hide samples shelf</string>
-    <string name="revanced_hide_samples_shelf_summary">Hides samples shelf in feed.</string>
-    <string name="revanced_hide_cast_button_title">Hide cast button</string>
-    <string name="revanced_hide_cast_button_summary">Hides cast button.</string>
+    <string name="revanced_hide_playlist_card_shelf_summary">Hides the playlist card shelf in the feed.</string>
+    <string name="revanced_hide_samples_shelf_title">Hide Samples shelf</string>
+    <string name="revanced_hide_samples_shelf_summary">Hides the Samples shelf in the feed.</string>
+    <string name="revanced_hide_cast_button_title">Hide Cast button</string>
+    <string name="revanced_hide_cast_button_summary">Hides the Cast button.</string>
     <string name="revanced_hide_category_bar_title">Hide category bar</string>
-    <string name="revanced_hide_category_bar_summary">Hides category bar.</string>
+    <string name="revanced_hide_category_bar_summary">Hides the category bar.</string>
     <string name="revanced_hide_floating_button_title">Hide floating button</string>
-    <string name="revanced_hide_floating_button_summary">Hides floating button in library.</string>
+    <string name="revanced_hide_floating_button_summary">Hides the floating button in the Library tab.</string>
     <string name="revanced_hide_tap_to_update_button_title">Hide \'Tap to update\' button</string>
-    <string name="revanced_hide_tap_to_update_button_summary">Hides \'Tap to update\' button.</string>
-    <string name="revanced_hide_history_button_title">Hide history button</string>
-    <string name="revanced_hide_history_button_summary">Hides history button in toolbar.</string>
-    <string name="revanced_hide_notification_button_title">Hide notification button</string>
-    <string name="revanced_hide_notification_button_summary">Hides notification button in toolbar.</string>
+    <string name="revanced_hide_tap_to_update_button_summary">Hides the \'Tap to update\' button.</string>
+    <string name="revanced_hide_history_button_title">Hide History button</string>
+    <string name="revanced_hide_history_button_summary">Hides the History button in the toolbar.</string>
+    <string name="revanced_hide_notification_button_title">Hide Notifications button</string>
+    <string name="revanced_hide_notification_button_summary">Hides the Notifications button in the toolbar.</string>
     <string name="revanced_hide_sound_search_button_title">Hide sound search button</string>
-    <string name="revanced_hide_sound_search_button_summary">Hides sound search button in search bar.</string>
+    <string name="revanced_hide_sound_search_button_summary">Hides the sound search button in the search bar.</string>
```

Comment:
Is it officially called `Sound` search button? When searching, some articles are calling it `Hum to search`. But in YouTube, it's represented as `Song` search.

Reply:
I'm pretty sure YouTube's development team and YouTube Music's development team don't communicate with each other

Official string from YouTube:
```
    <string name="try_song_search">Try song search</string>
    <string name="try_voice_search">Try voice search</string>
```

Official string from YouTube Music:
```
    <string name="accessibility_sound_search">Sound search</string>
    <string name="accessibility_voice_search">Voice search</string>
```